openedx/features/edly/utils.py

```diff
@@ -415,3 +416,25 @@ def get_marketing_link(marketing_urls, name):
     else:
         LOGGER.warning("Cannot find corresponding link for name: %s", name)
         return ''
+
+def is_course_org_same_as_site_org(site, course_id):
+    """
+    Check if site has same organization as course.
```

Comment:
Since in the method name `is_course_org_same_as_site_org`, `course_org` come before `site_org` so we can add method docscstring like:
`Check if the course organization matches with the site organization.`

Reply:
Done